src/components/views/PCIForm.js

```diff
@@ -0,0 +1,193 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { isEqual } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+import {
+  AccordionSet,
+  Button,
+  Col,
+  ExpandAllButton,
+  IconButton,
+  Layout,
+  Pane,
+  PaneFooter,
+  PaneMenu,
+  Paneset,
+  Row,
+  Spinner,
+} from '@folio/stripes/components';
+import { TitleManager } from '@folio/stripes/core';
+import stripesFinalForm from '@folio/stripes/final-form';
+import css from './PCIForm.css';
+
+import { PCIFormCoverage, PCIFormInfo } from '../EResourceSections';
+
+class PCIForm extends React.Component {
+  static propTypes = {
+    form: PropTypes.shape({
+      getRegisteredFields: PropTypes.func.isRequired,
+    }).isRequired,
+    handlers: PropTypes.PropTypes.shape({
+      onClose: PropTypes.func.isRequired,
+    }),
+    initialValues: PropTypes.object,
+    handleSubmit: PropTypes.func.isRequired,
+    pristine: PropTypes.bool,
+    submitting: PropTypes.bool,
+    values: PropTypes.object,
+  }
+
+  static defaultProps = {
+    initialValues: {},
+  }
+
+  state = {
+    sections: {
+      pciFormCoverage: true,
+    }
+  }
+
+  getSectionProps(id) {
+    const { form, values = {} } = this.props;
+
+    return {
+      form,
+      id,
+      onToggle: this.handleSectionToggle,
+      open: this.state.sections[id],
+      values,
+    };
+  }
+
+  handleSectionToggle = ({ id }) => {
+    this.setState((prevState) => ({
+      sections: {
+        ...prevState.sections,
+        [id]: !prevState.sections[id],
+      }
+    }));
+  }
+
+  handleAllSectionsToggle = (sections) => {
+    this.setState({ sections });
+  }
+
+  renderPaneFooter() {
+    const {
+      handlers,
+      handleSubmit,
+      pristine,
+      submitting,
+    } = this.props;
+
+    return (
+      <PaneFooter
+        renderEnd={(
+          <Button
+            buttonStyle="primary mega"
+            disabled={pristine || submitting}
+            id="clickable-update-pci"
+            marginBottom0
+            onClick={handleSubmit}
+            type="submit"
+          >
+            <FormattedMessage id="stripes-components.saveAndClose" />
+          </Button>
+        )}
+        renderStart={(
+          <Button
+            buttonStyle="default mega"
+            id="clickable-cancel"
+            marginBottom0
+            onClick={handlers.onClose}
+          >
+            <FormattedMessage id="stripes-components.cancel" />
+          </Button>
+        )}
+      />
+    );
+  }
+
+  renderFirstMenu() {
+    return (
+      <PaneMenu>
+        <FormattedMessage id="ui-agreements.pci.closeEdit">
+          {ariaLabel => (
+            <IconButton
+              aria-label={ariaLabel}
+              icon="times"
+              id="close-pci-form-button"
+              onClick={this.props.handlers.onClose}
+            />
+          )}
+        </FormattedMessage>
+      </PaneMenu>
+    );
+  }
+
+  renderLoadingPane = () => {
```

Comment:
This function is never called. If it's supposed to be, use `<LoadingView>` instead.